firmware/Components.cpp

```diff
@@ -163,11 +165,6 @@ void ServoSensor::position(float position)
   servo_legacy_.writeMicroseconds(command);
 }
 
-float ServoSensor::position()
-{
-  return position_;
-}
```

Comment:
Copy and paste error. I was putting back in the servo legacy code so that I could separate the pull requests. I must have pasted over the position method